### ISSUE TITLE
Ratio resetting

### ISSUE THREAD

**Issue by Dulanic** [binhex/arch-rtorrentvpn#49]
I am noticing that every few days I am seeing my ratio settings reset to default. Any reasoning this may be happening? I haven't noticed anything else so far, but will keep a look out. All my paths are bound.

**Comment by josjaf** [comment]
+1 I am having the same problem

**Comment by josjaf** [comment]
I also notice that torrents aren't obeying their ratios, they are stopped when I have manually started them. The ratio is stopping them for some reason

**Comment by Dulanic** [comment]
Found the issue is because the conf file is stored here which is not persistent:
/usr/share/webapps/rutorrent/plugins/ratio/conf.php

**Comment by binhex** [comment]
> Found the issue is because the conf file is stored here which is not persistent:
/usr/share/webapps/rutorrent/plugins/ratio/conf.php
I'd submit a PR, but I'm not sure if it's just a ready-me change for the volume mapping /usr/share/webapps/rutorrent/plugings to a mount point or if other things need to be added too.

This is not the issue, the file located at /usr/share/webapps/rutorrent/plugins/ratio/conf.php is for configuration of the plugin NOT for user generated configuration. User configuration of all plugins is located at /config/rutorrent/share/users/admin/settings/ratio.dat and will be persistent. 

So either /config is located on a non persistent media or there is a bug in the ratio plugin.

**Comment by Dulanic** [comment]
You're right, I'm not even sure where it does store the setting. I will keep an eye out if it resets again and then do further research.

**Comment by binhex** [comment]
> I'm not even sure where it does store the setting

/config/rutorrent/share/users/admin/settings/ratio.dat

**Comment by josjaf** [comment]
I'm fairly sure this is a problem with the ratio plugin then. @binhex Does the container clone the rutorrent repo from git? 

Also this container is absolutely fantastic. It basically removes my need for the old rtorrent vm aka pia tunnel and pfsense. So thanks for the hard work!
